packages/cms-cli/cmds/fetch.js

```diff
@@ -0,0 +1,72 @@
+const { getCwd } = require('@hubspot/cms-lib/path');
```

Comment:
The `/cmds` dir is the copied from Yargs' example
https://github.com/yargs/yargs/tree/master/example/cmds